.delivery/build-cookbook/recipes/syntax.rb

```diff
@@ -1,3 +1,5 @@
+return if (workflow_stage?('union') || workflow_stage?('rehearsal'))
```

Comment:
This is redundant as the `syntax` phase is not part of the `union` or `rehearsal` stages.